Source/JavaScriptCore/jit/ExecutableAllocator.cpp

```diff
@@ -1372,7 +1372,8 @@ void dumpJITMemory(const void* dst, const void* src, size_t size)
 
     static std::once_flag once;
     std::call_once(once, [] {
-        buffer = std::bit_cast<uint8_t*>(malloc(bufferSize));
+        // FIXME: <rdar://162872180> std::bit_cast does not provide a hint to malloc().
+        buffer = std::bit_cast<uint8_t*>(malloc(bufferSize * sizeof(*buffer)));
```

Comment:
Actually, this is not intentionally using `malloc`. So probably `LazyNeverDestroyed<Vector<uint8_t>>` would be fine. But as it is only used for CPU data collection (very specific use case for data collection), just changing it to this form is fine.